debezium-core/src/main/java/io/debezium/pipeline/signal/SignalProcessor.java

```diff
@@ -69,14 +72,18 @@ public SignalProcessor(Class<? extends SourceConnector> connector,
                            Offsets<P, O> previousOffsets) {
 
         this.connectorConfig = config;
-        this.signalChannelReaders = signalChannelReaders;
+        this.totalChannelReaders = signalChannelReaders;
```

Comment:
WDYT to have a all available channels in the `signalChannelReaders` variable and then introduce a `enabledChannelReades` the will contains the filtered list? I think will be more readable.

Reply:
Good idea, I will change it.